### ISSUE TITLE
Crash du système

### ISSUE THREAD

**Issue by GerardPaligot** [zestedesavoir/extensions-notificateurs#39]
Salut,

Je viens rapporter un bug ennuyant puisque l'extension fait littérelament crasher mon système et malheureusement, je ne vais pas pouvoir donner beaucoup d'information.

Procédure :

- Recevoir une notification.
- Cliquer dessus.
- Se rendre compte que le pannel n'est jamais ouvert et que le système est crashé.

Attention, le bug ne survient pas systématiquement. En fait, il est plutôt rare mais il arrive assez régulièrement pour que je vienne le rapporter directement sur le dépôt technique.

Environnement :

Système : OS X 10.9.5 (Mavericks).
Navigateur : Google Chrome Version 51.0.2704.106 (64-bit)

**Comment by viki53** [comment]
Hmm… tu pourrais détailler ce que tu entends par crasher ?

Tu as regardé le moniteur système, l'usage CPU & RAM ?

**Comment by GerardPaligot** [comment]
Le curseur de ma souris m'indique que mon système bosse sur un truc, je peux bouger le curseur mais je ne peux rien faire. Mon clavier ne répond plus et les cliques de ma souris non plus. Quand je ferme mon portable pour normalement le mettre en veille, il ne le fait pas. Je ne peux juste rien faire à part un force reboot.

**Comment by viki53** [comment]
Ah oui, c'est violent !

Tu aurais une autre extension qui ouvre une popup dans le même genre (style ChromeCast ou HTTPSEverywhere), pour comparer ?

**Comment by GerardPaligot** [comment]
Oui, j'ai Chromecast à côté mais ça arrive quand je clique sur notre petite clémentine seulement.

**Comment by viki53** [comment]
Et comme le système répond pas je suppose qu'inspecter la popup doit être impossible…

Ça va être sympa à reproduire et corriger, ça !

Tu as testé sur d'autres versions du système (OS X), avec la même machine ?

**Comment by GerardPaligot** [comment]
> Ça va être sympa à reproduire et corriger, ça !

Malheureusement. :/

> Tu as testé sur d'autres versions du système (OS X) et/ou du navigateur (Chrome), avec la même machine ?

Malheureusement, je n'ai pas d'autres versions d'OS X sous la main. Par contre, c'est récent comme bug. Je n'avais jamais eu ça avant quelques jours ou semaines. Peut-être une mise à jour de Chrome ?

**Comment by viki53** [comment]
Si c'est récent, ça peut effectivement être lié à Chrome… 

Faudrait que je refasse un tour du code et comparer avec la doc pour voir s'il y a pas des mises à jour à faire pour les extensions.

**Comment by vhf** [comment]
Ça pue le memory leak js ça. J'y jette un œil dans la semaine si j'oublie
pas. Sinon hésitez pas à me pinger ici.
